pkg/cafs/reader.go

```diff
@@ -14,6 +14,11 @@ import (
 	"github.com/oneconcern/datamon/pkg/storage"
 )
 
+type Reader interface {
```

Comment:
this type is added (1) for consistency with the `Writer` type and (2) to avoid the type assertion that was formerly present in `defaultFs.GetAt()`